### ISSUE TITLE
Request for support for Nano receiver

### ISSUE THREAD

**Issue by thymaro** [pwr/Solaar#419]
I am not a developer, so I don't know how to formulate my problem correctly (and, by extension, of course, I can't solve it myself or provide any form of suggestion as to what code to include in the master branch), sorry about that.

I would like to ask for inclusion of the following device:
nano receiver
ID: LZ137AX-DJ
M/N:C-U0008

I have installed solaar and rebooted my system, with unplugged and plugged  receiver, but it never shows up in the solaar paired devices list. I any more work on this wonderful piece of software planned?

**Comment by doctor64** [comment]
Please, can you provide usb id of your receiver? Just plug it in and issue lsusb command in terminal.

**Comment by thymaro** [comment]
**tl;dr Problem "solved". Close thread.**

I did as suggested, but the device wasn't even _visible_ to the machine, which made me suspicious. So I took another receiver from a working machine, plugged that in, which worked and got recognized. I plugged in the unrecognized receiver into the other machine and nothing happened. I think the new receiver is broken. My problem isn't solved, but you can't do anything about it. Thanks for this great piece of software and for getting back to me. Carry on!
